Kumi.Game/Screens/Play/Player.cs

```diff
@@ -17,6 +17,8 @@ namespace Kumi.Game.Screens.Play;
 
 public partial class Player : ScreenWithChartBackground
 {
+    private const float pass_threshold = 0.7f;
```

Comment:
This should probably be in the health processor, with a function saying if we've passed the threshold. We may want to change the threshold depending on difficulty.

Reply:
I think it's fine where it is for the time being.